packages/ui/src/components/va-collapse/VaCollapse.vue

```diff
@@ -207,14 +206,18 @@ export default class VaCollapse extends mixins(
   mounted () {
     this.getHeight()
 
+    this.setCollapseParams()
+
     this.mutationObserver = new MutationObserver(() => {
-      this.setCollapseParams()
+      setTimeout(() => this.setCollapseParams(), 1000 / 60)
```

Comment:
Why 1000/60? Very peculiar timing.

Reply:
16ms. I thought 1000 / 60fps will be much readable than 16ms.